[docs] Guides: sidebar navigation (#23352)
expo/expo: docs/pages/guides/overview.mdx

---
title: Overview
hideTOC: true
---

> This section contains information about the following process and services:

Reviewer: ```suggestion
This section contains information about the development with Expo and Expo Application Services (EAS):
```

> - **Development process**: This section contains guides about additional configurations and workflows that you may require during the development process to help you develop, deploy, and maintain your app. It contains in-depth information about [app config](/workflow/configuration/), [permissions](/guides/permissions/), [universal links](/guides/deep-linking/), [custom native code](/workflow/continuous-native-generation/), [web](/workflow/web/), and more.

Reviewer: ```suggestion
- **Development process**: Learn about additional configurations and workflows that you may require during the development process to help you develop, deploy, and maintain your app. It contains in-depth information about [app config](/workflow/configuration/), [permissions](/guides/permissions/), [universal links](/guides/deep-linking/), [custom native code](/workflow/continuous-native-generation/), [web](/workflow/web/), and more.
```

> - **Expo Application Services (EAS)**: Deeply integrated cloud services for Expo and React Native apps, from the team behind Expo. There are different services available such as [EAS Build](/build/introduction/), [EAS Submit](/submit/introduction/), [EAS Update](/eas-update/introduction/), [EAS Metadata](/eas/metadata/) and [EAS Insights](/eas-insights/introduction/).

Reviewer: ```suggestion
- **Expo Application Services (EAS)**: Deeply integrated cloud services for Expo and React Native apps, from the team behind Expo. Learn about [EAS Build](/build/introduction/), [EAS Submit](/submit/introduction/), [EAS Update](/eas-update/introduction/), [EAS Metadata](/eas/metadata/), and [EAS Insights](/eas-insights/introduction/).
```

Adding the oxford comma too, since that's in our styleguide 🤓
